src/main/java/ch/sportchef/business/authentication/entity/Challenge.java

```diff
@@ -0,0 +1,21 @@
+package ch.sportchef.business.authentication.entity;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+
+@Entity
+@Data
+public class Challenge {
+
+    private final String challenge;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'challenge' could be replaced by a local variable.](https://www.codacy.com/app/McPringle/sportchef/file/2485993592/issues/source?bid=3149602&fileBranchId=3265763#l11)